### PR TITLE
React  Todolist

### PR DESCRIPTION
**Pull request by Santiago220991**
# Tasks done:

- Initialized a new React project.
- Added a feature to add a new task.
- Added a feature to delete a task.
- Added a feature to edit a task.
- Added a feature to mark up completed tasks.
- Added styling to the project.

**Note: The project is not deployed using Github pages yet.**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,62 @@
...
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+- React
+
+## Live Demo
+
+[Javascript Capstone Live Demo](https://santiago220991.github.io/React-tutorial/dist)
```
Comment (SebastianSolisFenger) [inline README.md]: [OPTIONAL] => Kindly, check the live demo works correctly.

**Review by SebastianSolisFenger — APPROVED** [review]
STATUS: APPROVED 🟢 👏
Hi @Santiago220991,

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

Some highlights for you
Correct Gitflow practice ✔️
Professional README ✔️
Clean codebase ✔️
well done!!

Optional suggestions:
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

File: README.md (hunk 1 above)
Comment (Santiago220991) [reply README.md]: Thanks. 👍

**Comment by Santiago220991** [comment]
The project has already been deployed. Thanks for the suggestions.  👍

**Pull request merged**